card/mod/05_standard/set/self/signin.rb

```diff
@@ -96,6 +95,10 @@ def default_core_args args={}
   end  
 end
 
+event :signin_success, :after=>:signin do
+  abort :success
+end
+
```

Comment:
Yeah, the latter is a little more odd looking, but the independence of ordering is a plus.